xknx/devices/climate.py

```diff
@@ -168,6 +170,15 @@ def __init__(
 
         self.mode = mode
 
+        self.humidity = RemoteValueHumidity(
```

Comment:
I think this could just use `RemoteValueNumeric` with a constant `value_type` instead of introducing a new RemoteValue class that's mostly single-used boilerplate.

(I was planning to add support for passing `type[DPTBase]` classes to a RemoteValue `__init__` additional to `value_type: int | str` someday. This would spare the value_type lookup then.)

Reply:
You can find examples in the Light class. https://github.com/XKNX/xknx/blob/7f6bcfb6cc0dfd60c0bfc813e7ec5f39147a0d4f/xknx/devices/light.py#L212
Not sure why RemoteValueTemp still exists 🙃